### PR TITLE
DX: cleanup - remove special treatment for PHP <5.6

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: tests/Fixer/ClassNotation/SelfAccessorFixerTest.php [hunk 1]
```diff
@@ -83,7 +83,6 @@ public function provideFixCases()
                 '<?php class Foo { function bar() { new Foo\Baz(); } }',
             ],
             [
-                // PHP < 5.4 compatibility: "self" is not available in closures
                 '<?php class Foo { function bar() { function ($a = Foo::BAZ) { new Foo(); }; } }',
```
Comment (keradus) [inline tests/Fixer/ClassNotation/SelfAccessorFixerTest.php]: so, `self::BAZ` ?

Comment (kubawerlos) [reply tests/Fixer/ClassNotation/SelfAccessorFixerTest.php]: As an expected?

Comment (kubawerlos) [reply tests/Fixer/ClassNotation/SelfAccessorFixerTest.php]: Oki doki, updated test case and fixer.

**Comment by SpacePossum** [comment]
Thanks @kubawerlos.

**Pull request merged**